manifests/init.pp

```diff
@@ -58,6 +58,7 @@
   $dbpassword                                                       = 'mypassword',
   $dbserver                                                         = 'localhost',
   $dbname                                                           = 'jira',
+  Boolean $oracle_sid                                                  = true,
```

Comment:
can you please update the alignment of `=` to match the others?

Reply:
Ok done